app/views/discovered_mailer/_discovered_host.html.erb

```diff
@@ -2,7 +2,7 @@
 <td style="<%= td_style%>">
   <%= link_to host, discovered_host_path(:id => host, :host => @url.host, :port => @url.port, :only_path => false, :protocol => @url.scheme) %>
 </td>
-<td style="<%= td_style%>"><%= host.model_name %></td>
+<td style="<%= td_style%>"><%= host.try(:model_name) || 'N/A' %></td>
```

Comment:
> You should be using .hardware_model_name as it was renamed in Foreman core, otherwise this will always result in 'N/A'.  See http://projects.theforeman.org/issues/13440 and https://groups.google.com/d/msg/foreman-dev/CLhSVaZsi1g/p9yMwJhMHAAJ.

I want to keep the fix and possible refactoring separate for easier
cherry picking. I already filed an issue for the refactoring task.
Thanks.

-- 
Later,
 Lukas #lzap Zapletal